server/api/archive.go

```diff
@@ -0,0 +1,144 @@
+package api
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/mattermost/focalboard/server/model"
+	"github.com/mattermost/focalboard/server/services/audit"
+)
+
+func (a *API) handleArchiveExport(w http.ResponseWriter, r *http.Request) {
+	// swagger:operation GET /api/v1/workspaces/{workspaceID}/archive/export archiveExport
+	//
+	// Exports an archive of all blocks for one or more boards. If root_id is provided then
+	// only that baord will be exported, otherwise all boards in the workspace.
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: workspaceID
+	//   in: path
+	//   description: Workspace ID
+	//   required: true
+	//   type: string
+	// - name: root_id
+	//   in: path
+	//   description: Root id (board) to export
+	//   required: false
+	//   type: string
+	// security:
+	// - BearerAuth: []
+	// responses:
+	//   '200':
+	//     description: success
+	//     schema:
+	//       type: array
+	//       items:
```

Comment:
This is not true, the expected result is a binary file.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/2201/commits/4635eb5b6b79391af0ab7f7ecc008d3a7d7704a8